ccd-definition/CaseTypeTab/1_CaseHistory.json

```diff
@@ -8,5 +8,27 @@
     "TabDisplayOrder": 1,
     "CaseFieldID": "caseHistory",
     "TabFieldDisplayOrder": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "Channel": "CaseWorker",
+    "TabID": "CaseHistory",
+    "TabLabel": "History",
+    "TabDisplayOrder": 1,
+    "CaseFieldID": "familyManCaseNumber",
+    "TabFieldDisplayOrder": 2,
+    "FieldShowCondition": "familyManCaseNumber = \"DO_NOT_SHOW\""
```

Comment:
not sure if it is possible in tab, but maybe "Comment" attribute can be added (possible in filed definition)